Services/UI/OJS.Services.Ui.Business/Validation/ValidationMessages.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace OJS.Services.Ui.Business.Validation;
+
+public static class ValidationMessages
+{
+    public const string ContestNotFound = "Contest not found!";
```

Comment:
create a nested class to hold the messages for Contest

Reply:
Done.